Krimirundgang/Krimirundgang.csproj

```diff
@@ -84,6 +109,7 @@
     <Folder Include="Resources\drawable\drawable-xhdpi\" />
     <Folder Include="Resources\drawable\drawable-xxhdpi\" />
     <Folder Include="Resources\drawable\drawable-xxxhdpi\" />
+    <Folder Include="Resources\raw\" />
```

Comment:
The folder raw is empty. We don't use it anymore. It can be removed fromt he Project